source/includes/_authentication.md

```diff
@@ -20,16 +20,25 @@ mobile devices or running in external servers. The supported OAuth flows:
 
 Once a user is authorized, **every** API request **must** include a valid access token.
 
-**Before proceeding:** pick the OAuth flow that is suitable for your business and skip the flows
-that do not apply to your account in this guide.
+<aside class="warning">
+  <strong>Before proceeding:</strong> pick the OAuth flow that is suitable for your business and skip the flows that do not apply to your account in this guide.
+</aside>
+
+
+
 
 The following OAuth 2.0 endpoints are available under `https://connect.remotelock.com/`.
+
 These are needed for OAuth2 client library you'll be using:
 
-        GET       /oauth/authorize
-        POST      /oauth/token
+<div class="center-column"></div>
```

Comment:
I'm not sure what you mean by center column - can you post a screenshot of what it looks like?

Reply:
https://azurapi.github.io/

my doc has the implementation but new updates from slate prevented from happening

Reply:
Sure, I've added to the description what it looks like without this.

And while I was preparing the screenshot, I found that just a `<div />` could resolve part of the problems that I was facing, that was trying to align text with examples.

![Screen Shot 2020-06-23 at 09 18 59](https://user-images.githubusercontent.com/29945/85402794-aafe3480-b532-11ea-8ca6-3849d7558203.png)

Reply:
I think I like the `<div />` approach better -- having it all in the center column feels like we're wasting that space on the right, which is one of the nice things of slate

Reply:
Yeah me too, I've tried different workarounds before but found this by accident. 😄 

I will need to verify again to check if works in all cases because some parts have a lot of blocks in sequence and could be hard to read.

And just to clarify, this is mainly used outside endpoints docs, I think just one endpoint has this "middle column examples".